### PR TITLE
Prometheus metric for backup version shows major and minor version

### PR DESCRIPTION
**Pull request by iuryr**
This pull request implements a change to the prometheus metric `pgmoneta_backup_server_version` in order to show major and minor versions separately.

- `src/libpgmoneta/info.c` changes populates `backup->minor_version` with pertinent information
- `src/libpgmoneta/prometheus.c` changes adjusts the display of the metric in order to show aforementioned data and also changes documentation displayed in `home_page` function

### CODE REVIEW

**Comment by jesperpedersen** [comment]
You need to add your name to `AUTHORS`.

Then squash, rebase and force push

**Comment by iuryr** [comment]
> You need to add your name to `AUTHORS`.
> 
> Then squash, rebase and force push

Done!

**Comment by jesperpedersen** [comment]
`pgmoneta_backup_version` isn't documented on the main page, can you make sure that all metrics are ?

Do `git grep "#TYPE"` and make sure they are in the `home_page()` function.

Squash, rebase and force push.

Thanks !

**Comment by iuryr** [comment]
@jesperpedersen The changes you asked in https://github.com/pgmoneta/pgmoneta/pull/240#issuecomment-2039444518 are implemented, please review to see if I let something slip by.

I edited https://github.com/pgmoneta/pgmoneta/pull/240#issue-2224099427 to include the changes in src/libpgmoneta/prometheus.c

Also, before the changes above, I synced my fork with pgmoneta repo and did a `git pull` so the commits of the main branch are now included in this PR. I don't know if this messes with some workflow somehow. If it does, please advise of how I can fix it.

Thank you!

**Comment by iuryr** [comment]
Yeah, just had to rebase and drop the commits that got included because of the pull. Only one commit in this PR now. Phew :sweat_smile:

**Comment by iuryr** [comment]
aaaand I just realized that you asked me to check that all other metrics are documented in `home_page` function. Sorry. Will check and update the PR later!

File: src/libpgmoneta/prometheus.c [hunk 1]
```diff
@@ -432,6 +432,29 @@ home_page(int client_fd)
...
+   data = pgmoneta_append(data, "        <td>name</td>\n");
+   data = pgmoneta_append(data, "        <td>The identifier for the server</td>\n");
+   data = pgmoneta_append(data, "      </tr>\n");
+   data = pgmoneta_append(data, "      <tr>\n");
+   data = pgmoneta_append(data, "        <td>label</td>\n");
+   data = pgmoneta_append(data, "        <td>The backup label</td>\n");
+   data = pgmoneta_append(data, "      </tr>\n");
+   data = pgmoneta_append(data, "      <tr>\n");
+   data = pgmoneta_append(data, "        <td>major</td>\n");
+   data = pgmoneta_append(data, "        <td>The backup postgresql major version</td>\n");
```
Comment (jesperpedersen) [inline src/libpgmoneta/prometheus.c:449]: Its `PostgreSQL`

File: src/libpgmoneta/prometheus.c [hunk 2]
```diff
@@ -432,6 +432,29 @@ home_page(int client_fd)
...
+   data = pgmoneta_append(data, "      </tr>\n");
+   data = pgmoneta_append(data, "      <tr>\n");
+   data = pgmoneta_append(data, "        <td>minor</td>\n");
+   data = pgmoneta_append(data, "        <td>The backup postgresql minor version</td>\n");
```
Comment (jesperpedersen) [inline src/libpgmoneta/prometheus.c:453]: Same

**Review by jesperpedersen — CHANGES_REQUESTED** [review]

**Comment by jesperpedersen** [comment]
There is still something wrong with your rebase...

https://www.git-tower.com/learn/git/faq/git-squash
https://github.com/progit/progit2/releases/tag/2.1.425

**Comment by jesperpedersen** [comment]
You can use the same PR, just force push

https://github.com/pgmoneta/pgmoneta/blob/main/doc/DEVELOPERS.md#basic-git-guide

**Comment by iuryr** [comment]
I discarded my commit only on my remote fork of pgmoneta and Github automatically closed the PR (I did not know it would do that). I'm working on the changes now and will reuse this PR, as avised. Thank you!

**Comment by iuryr** [comment]
Reopening PR with (hopefully) no rebase problems and with all prometheus metrics documented in `home_page()`.

**Comment by jesperpedersen** [comment]
Merged.

Thanks for your contribution !

**Comment by iuryr** [comment]
@jesperpedersen I noticed a typo in `prometheus.c`. A  `td` tag that is not properly closed in line 294 in the file version of this commit. Should I reopen the PR and fix it?

**Comment by jesperpedersen** [comment]
Yes

**Comment by jesperpedersen** [comment]
You can use https://validator.w3.org/

**Comment by iuryr** [comment]
Jesper, 

It seems that I cannot reuse this PR to submit the fix, since it has already been merged. I will open another PR and reference this one, if that's ok with you. Please let me know. Thanks

**Comment by jesperpedersen** [comment]
Yeah, that is ok

**Pull request merged**